p2p/net/mock/mock_stream.go

```diff
@@ -30,3 +51,55 @@ func (s *stream) Close() error {
 func (s *stream) Conn() inet.Conn {
 	return s.conn
 }
+
+// transport will grab message arrival times, wait until that time, and
+// then write the message out when it is scheduled to arrive
+func (s *stream) transport() {
+	bufsize := 256
```

Comment:
the only purpose the buffer serves is to coalesce small messages into a single transmission. Similar to how the network ring buffer works, It could be raised, but I dont beleive it will significantly affect performance in our tests.